cmd/nodepool/core/create.go

```diff
@@ -0,0 +1,116 @@
+package core
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/cmd/util"
+	hyperapi "github.com/openshift/hypershift/support/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type CreateNodePoolOptions struct {
+	Name         string
+	Namespace    string
+	ClusterName  string
+	NodeCount    int32
+	ReleaseImage string
+	Render       bool
+}
+
+// ApplyPlatformSpecifics can be used to create platform specific values as well as enriching the fixure with additional values
```

Comment:
This comment seems to refer to a different function.

Reply:
Fixed